### ISSUE TITLE
{file_name} is not working

### ISSUE THREAD

**Issue by jefflima1** [univrsal/tuna#192]
Using a VLC Video source, it is not possible to detect the name of the file being played, it only works as if the {title} was configured

![image](https://github.com/univrsal/tuna/assets/106811319/350a4c46-14d7-45e4-9678-08ebf60a66e9)

![image](https://github.com/univrsal/tuna/assets/106811319/94e03f3a-08bc-4f30-aaf0-fea66c84410b)

**Comment by DriesMagnus** [comment]
I hope this can get fixed at some point, it's rather annoying to have to strip some audio files of their metadata just so it gets the file name using {title}

**Comment by univrsal** [comment]
The obs vlc source only supports retrieving the title of the currently playing file, which is the file name if the file doesn't contain the title metadata field. That's just how libvlc works.

For it to return the actual file name someone would have to submit a patch to obs which could take quite a bit until it would make it into an obs release.

Technically the VLC source being marked as supporting the file name is just wrong, the field is never set in the code.

**Comment by DriesMagnus** [comment]
Ah okay, in that case, unlucky!
Thanks for clarifying!

Is it possible to have some sort of warning in the plugin for this? So people aren't confused on it anymore.
Thank you for making this amazing plugin 🔥

**Comment by univrsal** [comment]
Yeah, if you use an unsupported format specifier it shows a warning. It just doesn't show it right now because the source is incorrectly configured to claim that it supports {file_name}. I've changed that now in 9903887
